docs/ref/contrib/gis/forms-api.txt

```diff
@@ -96,6 +96,14 @@ Widget attributes
 GeoDjango widgets are template-based, so their attributes are mostly different
 from other Django widget attributes.
 
+.. attribute:: BaseGeometryWidget.base_layer
```

Comment:
We might want a release note for this new attribute

Reply:
Good point, adding it under `django.contrib.gis`.